source/common/json/config_schemas.cc

```diff
@@ -21,6 +21,12 @@ const std::string Json::Schema::LISTENER_SCHEMA(R"EOF(
             "type" : "array",
             "items" : {
               "type" : "string"
+            },
+            "session_ticket_key_paths": {
+              "type" : "array",
+              "items" : {
+                "type" : "string"
+              }
```

Comment:
You've added this one level too deep (right now it's part of `verify_subject_alt_name`), breaking the whole schema.

Reply:
Yuck, that must have happened during a rebase/merge, and somehow I missed it.